package.json

```diff
@@ -3,26 +3,35 @@
   "version": "1.0.0",
   "private": true,
   "scripts": {
-    "firstbuild": "npm install && fable",
-    "prebuild": "npm install",
-    "build": "fable --watch"
+    "build": "webpack",
+    "start": "webpack-dev-server",
```

Comment:
Ah, Fable.Templates.Elmish.React? That one is for the web, we'll need a new one for the native once you figured out what goes where :)